variable/include/scipp/variable/variable_comparison.h

```diff
@@ -13,8 +14,19 @@
 
 namespace scipp::variable {
 
-/// Tests if the unit, values (and variances where appropriate) of two
-/// Variables are within an absolute tolerance.
+SCIPP_VARIABLE_EXPORT Variable is_less(const VariableConstView &x,
```

Comment:
Why not same name as on the Python side? I think this would just lead to confusion and suggest to use the Numpy name also on the C++ side.

Reply:
removed `is_ `